src/modules/Notification/Notification.story.mdx

```diff
@@ -0,0 +1,321 @@
+import Notification from "./Notification";
+
+<Meta title="Components| Notification" />
+
+# Notification
+
+The `Notification` component can be used independently or in a `NotificationStack`. It should be used to convey meaningful but concise information to the user and should be easily noticeable and understood.
+As of now, the `Notification`'s content/body can only be a string.
+
+<Preview>
+  <Story name="Notification">
+    <Notification>
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+    </Notification>
+  </Story>
+</Preview>
+
+### Notification w/ title
+
+<Preview>
+  <Story name="Notification with title">
+    <Notification title="Title">
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+    </Notification>
+  </Story>
+</Preview>
+
+### Notification w/ icon
+
+<Preview>
+  <Story name="Notification with icon">
+    <Notification icon="check">Something good happened!</Notification>
+  </Story>
+</Preview>
+
+### Example of a custom success notification
+
+<Preview>
+  <Story name="Custom notification example">
+    <div style={{ display: "flex ", flexDirection: "column" }}>
+      Variation with icon:
+      <SuccessNotification icon="check">
+        Something good happened!
+      </SuccessNotification>
+      <br />
+      Variation with title:
+      <SuccessNotification title="Success!">
+        Something good happened!
+      </SuccessNotification>
+    </div>
+  </Story>
+</Preview>
+
+### Available props
+
+<table width="100%">
+  <thead>
+    <tr>
+      <th>Prop</th>
+      <th>Type</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>children</td>
+      <td>string</td>
+      <td>The text to be displayed in the Notification body</td>
+    </tr>
+    <tr>
+      <td>icon (optional)</td>
+      <td>string</td>
+      <td>
+        The name of the icon to be shown.
+        <br />
+        If this prop is passed, an icon will be displayed on the left of the
+        Notification body.
+      </td>
+    </tr>
+    <tr>
+      <td>title (optional)</td>
+      <td>string</td>
+      <td>
+        The title of the Notification
+        <br />
+        If this prop is passed, the Notification will have a title with the
+        given content.
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+### Extendable styles
+
+<table width="100%">
+  <thead>
+    <tr>
+      <th>Style Name</th>
+      <th>@selectors</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>icon</td>
+      <td></td>
+      <td>Style of the icon that precedes the text</td>
+    </tr>
+    <tr>
+      <td>text</td>
+      <td></td>
+      <td>Style of the Notification text</td>
+    </tr>
+    <tr>
+      <td>title</td>
+      <td></td>
+      <td>Style of the Notification title</td>
+    </tr>
+    <tr>
+      <td>wrapper</td>
+      <td></td>
+      <td>Style of the wrapper element</td>
+    </tr>
+    <tr>
+      <td></td>
+      <td>.enter</td>
```

Comment:
A more detailed description of these 4 selectors would be cool if you guys have any suggestions.

Reply:
I think it's understandable as it is, it follows the pattern of animations on React